flink-core/src/main/java/org/apache/flink/api/common/time/Time.java

```diff
@@ -32,10 +32,10 @@
 /**
  * The definition of a time interval. Similar to a simpler version of {@link java.time.Duration}.
  *
- * @deprecated Use {@link Duration}
+ * @deprecated Use {@link Duration}. Time needs to be kept until all external libraries remove this.
  */
 @Deprecated
-@PublicEvolving
+@Internal
```

Comment:
This is required because pyflink tests against the Kafka connector which still uses this class in the deprecated [FlinkKafkaProducer:183](https://github.com/apache/flink-connector-kafka/blob/818d1fdedaad63631eab5d44ec90c748cfcf299f/flink-connector-kafka/src/main/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaProducer.java#L183)

Reply:
It was concluded that we should disable affected tests for now (see [2.0 release sync logs from Sep 11, 2024](https://cwiki.apache.org/confluence/display/FLINK/2.0+Release#id-2.0Release-09/11/24))